PC02_Graffiti_COOPER-SANSONE.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Andrew Cooper-Sansone (9/4/2020)
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
```

Comment:
Remove Dr. Z's code from here. Then, add a label showing that this is where you will define variables. Then, add in the other variables you will need.
```suggestion
# Defining variables:

panel = Screen()
w = 750 # width of panel
h = 750 # height of panel
image = "Bezos.gif"

BezosTurt = Turtle()

left_eye = -15, 110
right_eye = 40, 120
laserTarget = 750, 100
```